example/inventory.rb

```diff
@@ -18,13 +18,13 @@
 
 class Product < BaseObject
   extend_type
-  key fields: 'upc'
+  key fields: :upc
 
   field :upc, String, null: false, external: true
   field :weight, Int, null: true, external: true
   field :price, Int, null: true, external: true
   field :in_stock, Boolean, null: true
-  field :shipping_estimate, Int, null: true, requires: { fields: 'price weight' }
+  field :shipping_estimate, Int, null: true, requires: { fields: %i[price weight] }
```

Comment:
I'd prefer not to use the `%` notation to be consistent with the documentation in some other places.

```suggestion
  field :shipping_estimate, Int, null: true, requires: { fields: [:price :weight] }
```

Reply:
I'm not a fan of readability of the `%i` notation either 🙂  This project uses the default Rubocop configuration for the `Style/SymbolArray` cop which requires it though. I'm happy to change the Rubocop configuration if you want.

Reply:
Ah.. Sorry, I did not notice the Rubocop config. I'll approve this change as is, and we can separately discuss what we want to do about styles.